modules/web/src/app/settings/admin/applications/component.ts

```diff
@@ -150,7 +150,6 @@ export class ApplicationsComponent implements OnInit, OnChanges {
           const index = this.applications.findIndex(app => app.name === updatedApplication.name);
           if (index !== -1) {
             this.applications[index] = updatedApplication;
-            this.dataSource.data = [...this.applications];
```

Comment:
Removing this altogether because updating it causes the whole table to re-render and flickers the toggles.